widgets/src/google/Common.js

```diff
@@ -39,7 +43,52 @@
+    Common.prototype.publish("hAxisDirection", 1, "number", "The direction in which the values along the horizontal axis grow. Specify -1 to reverse the order of the values.");
+    Common.prototype.publish("hAxisGridlinesCount", 5, "number", "The number of horizontal gridlines between two regular gridlines");
+    Common.prototype.publish("hAxisGridlinesColor", "#CCC", "html-color", "The color of the horizontal gridlines inside the chart area");
+    Common.prototype.publish("hAxisMinorGridlinesCount", 0, "number", "The number of horizontal minor gridlines between two regular gridlines");
+    Common.prototype.publish("hAxisMinorGridlinesColor", "#FFFFFF", "html-color", "The color of the horizontal minor gridlines inside the chart area");
+    Common.prototype.publish("hAxisLogScale", false, "boolean", "Makes horizontal axis a log scale");
+    Common.prototype.publish("hAxisTextPosition", "out", "set", "Position of the horizontal axis text, relative to the chart area", ["out","in","none"]);
+    Common.prototype.publish("hAxisTitle", "", "string", "Specifies a title for the horizontal axis");
+    Common.prototype.publish("hAxisMaxValue", null, "number", "Moves the max value of the horizontal axis to the specified value");
+    Common.prototype.publish("hAxisMinValue", null, "number", "Moves the min value of the horizontal axis to the specified value");
+    
 
+    Common.prototype.publish("vAxisBaseline", null, "number", "The baseline for the horizontal axis");
+    Common.prototype.publish("vAxisBaselineColor", "#000000", "html-color", "Specifies the color of the baseline for the vertical axis");
+    Common.prototype.publish("vAxisDirection", 1, "number", "The direction in which the values along the vertical axis grow. Specify -1 to reverse the order of the values.");
+    Common.prototype.publish("vAxisGridlinesCount", 5, "number", "The number of vertical gridlines between two regular gridlines");
+    Common.prototype.publish("vAxisGridlinesColor", "#CCC", "html-color", "The color of the vertical gridlines inside the chart area");
+    Common.prototype.publish("vAxisMinorGridlinesCount", 0, "number", "The number of vertical minor gridlines between two regular gridlines");
+    Common.prototype.publish("vAxisMinorGridlinesColor", "#FFFFFF", "html-color", "The color of the vertical minor gridlines inside the chart area");
+    Common.prototype.publish("vAxisLogScale", false, "boolean", "Makes vertical axis a log scale");
+    Common.prototype.publish("vAxisTextPosition", "out", "set", "Position of the vertical axis text, relative to the chart area", ["out","in","none"]);
+    Common.prototype.publish("vAxisTitle", "", "string", "Specifies a title for the vertical axis");
+    Common.prototype.publish("vAxisMaxValue", null, "number", "Moves the max value of the vertical axis to the specified value");
+    Common.prototype.publish("vAxisMinValue", null, "number", "Moves the min value of the vertical axis to the specified value");
+    
```

Comment:
the ones that i believe this is relevant too are already set to null, the rest are the correct values for the default value across multiple charts